example/simple_fetch_content/CMakeLists.txt

```
@@ -11,16 +11,28 @@ FetchContent_Declare(
   # GIT_REPOSITORY https://github.com/GridTools/cpp_bindgen.git
   # GIT_TAG        master # consider replacing master by a tagged version
 )
-# FetchContent_MakeAvailable(cpp_bindgen) for CMake >= 3.14, otherwise:
-FetchContent_GetProperties(cpp_bindgen)
-if(NOT cpp_bindgen_POPULATED)
-  FetchContent_Populate(cpp_bindgen)
-  add_subdirectory(${cpp_bindgen_SOURCE_DIR} ${cpp_bindgen_BINARY_DIR})
-endif()
+FetchContent_MakeAvailable(cpp_bindgen)
 
 # 2) create a library with bindings. This will generate the files simple.h and simple.f90 which can be included within C and Fortran. In CMake you can use them by linking against `simple_c`or `simple_fortran`.
 bindgen_add_library(simple SOURCES simple.cpp)
 
 # 3) link the generated library to a fortran executable
 add_executable(driver driver.f90)
 target_link_libraries(driver simple_fortran)
+
+# 4) optional: demonstrates installing the library with CMake
+# installs general cpp_bindgen targets
+install_cpp_bindgen_targets(
+  EXPORT simple_fortran_targets
+  Fortran_MODULE_DESTINATION include
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  )
+# install your library (simple) and the generated bindings library (simple_fortran) as usual
+install(
+  TARGETS simple simple_fortran
+  EXPORT simple_fortran_targets
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  )
+install(EXPORT simple_fortran_targets DESTINATION cmake NAMESPACE simple_fortran::)
```

```suggestion
install(EXPORT simple_fortran_targets DESTINATION cmake NAMESPACE SimpleFortran::)
```
I think that's the most common convention